fdm-app/app/components/custom/fertilizer-applications/formschema.tsx

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod"
+
+export const FormSchema = z.object({
+    p_app_amount: z.coerce
+        .number({
+            required_error: "Hoeveelheid is verplicht",
+            invalid_type_error: "Hoeveelheid moet een getal zijn",
+        })
+        .positive({
+            message: "Hoeveelheid moet positief zijn",
+        })
+        .finite({
+            message: "Hoeveelheid moet een geheel getal zijn",
+        })
+        .safe({
+            message: "Hoeveelheid moet een safe getal zijn",
+        }),
```

Comment:
what is "safe"?

Reply:
Seems not be needed as the value should already be positive and a finite number. I will remove this